qpc/cred/list.py

```diff
@@ -53,8 +54,20 @@ def _build_req_params(self):
 
     def _handle_response_success(self):
         json_data = self.response.json()
-        if json_data == []:
+        count = json_data.get('count', 0)
+        if count == 0:
             print(_(messages.CRED_LIST_NO_CREDS))
         else:
-            data = pretty_print(json_data)
+            data = pretty_print(json_data.get('results', []))
             print(data)
+
+        if json_data.get('next'):
+            next_link = json_data.get('next')
+            params = urlparse.parse_qs(urlparse.urlparse(next_link).query)
+            page = params.get('page', ['1'])[0]
+            if self.req_params:
+                self.req_params['page'] = page
+            else:
+                self.req_params = {'page': page}
+            input('Press enter to see the next set of results.')
```

Comment:
Does the man page need to describe this?

Reply:
I don’t believe so.